src/Http/RequestFactory.php

```diff
@@ -185,24 +187,34 @@ public function createHttpRequest()
 			}
 		}
 
+		$remoteAddr = !empty($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : NULL;
+		$remoteHost = !empty($_SERVER['REMOTE_HOST']) ? $_SERVER['REMOTE_HOST'] : NULL;
```

Comment:
Why change `isset` to `!empty`? Isset worked OK for years, is faster to execute, can be in PHP 7 rewritten with `??` and does not consider `0` as empty.